h/static/styles/partials-v2/_search.scss

```diff
@@ -106,7 +106,7 @@
 // Card displaying stats about a group of annotations in search results
 .search-bucket-stats {
   @include font-normal;
-  width: 210px;
+  width: 260px;
```

Comment:
Ideally we want the annotation card to have as much space as it needs up to 500px and then assign the leftover space to the stats box. In testing this again just now it looks like there have been some regressions at smaller screen sizes. For the moment I would just suggest changing this to `max-width` so that the box can be resized smaller than 260px if necessary.